Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -589,6 +584,12 @@ def mirror_investigation():
 
         send_slack_request_sync(CLIENT, 'chat.postMessage', body=body)
 
+    if kick_admin:
+        body = {
+            'channel': conversation_id
+        }
+        send_slack_request_sync(CLIENT, 'conversations.leave', body=body)
+
```

Comment:
Moved from line 575 and now will kick the admin after sending the first message.